docker-compose.yml

```diff
@@ -7,9 +7,7 @@ services:
       - MYSQL_USER=remo
       - MYSQL_PASSWORD=remo
       - MYSQL_ROOT_PASSWORD=root
-    entrypoint:
-      - docker-entrypoint.sh
-      - --max_allowed_packet=2M
+    command: mysqld --max_allowed_packet=64M
```

Comment:
I dont know the reason of increasing the ``max_allowed_packet``. Can you please explain?
moreover, its a good way to do it by the ``docker-entrypoint.sh``
We have already increased it to `2M`, if you need change, you can just change the value!